api/services.go

```diff
@@ -140,7 +140,7 @@ func (s *Service) Canonicalize(t *Task, tg *TaskGroup, job *Job) {
 
 // ConsulConnect represents a Consul Connect jobspec stanza.
 type ConsulConnect struct {
-	Native         bool
```

Comment:
Did Native work at all before? Just want to make sure we don't break jobfiles without good reason